opm/autodiff/BlackoilModelParameters.hpp

```diff
@@ -91,6 +91,9 @@ namespace Opm
         /// The file name of the deck
         std::string deck_file_name_;
 
+        // Whether to add influences of wells between cells to the matrix
```

Comment:
I do not understand the "between cells" comment?

Reply:
The sure complement introduces new connections (between cells) to the matrix.